docs/manual.adoc

```diff
@@ -0,0 +1,50 @@
+= Asciidoctor.js CLI User manual
+
+== Integration with Asciidoctor Reveal.js
+
+In this section, you'll learn:
+
+* How to install the required dependencies.
+* What attributes you must define.
+* How to convert an AsciiDoc file to a reveal.js presentation.
+
+=== Prerequisites
+
+We assume that you already have a fully working version of Asciidoctor.js CLI installed and available in your `PATH`.
+If this is not the case, please read the https://github.com/asciidoctor/asciidoctor-cli.js[installation instructions].
+
+To convert an AsciiDoc file to a reveal.js presentation, we will need to install the following module:
+
+ $ yarn add asciidoctor-reveal.js
+
+Once this module is installed, we create a file named `presentation.adoc` with the following content:
+
+.presentation.adoc 
+```adoc
+= Presentation title
+
+== First slide
+
+* Hello
+* World
+
+== Second slide
+
+With some *fascinating* content!
+```
+
+The next step is to convert this file to a reveal.js presentation using the `asciidoctorjs` command line:
+
+[source,sh]
+----
+asciidoctorjs presentation.adoc \
+  -r asciidoctor-reveal.js \ // <1>
+  -b revealjs \ // <2>
```

Comment:
@mojavelinux Do you think we should automatically `require('asciidoctor-reveal.js')`  and set the `revealjsdir` attribute to `node_modules/reveal.js@` ?

Reply:
I don't think so. We shouldn't require additional converters by default. It should be explicit. However, I do think we could come up with a way to auto-set the revealjsdir attribute (either here or in the converter somehow). That would certainly be a nice convenience for the user.

Reply:
I meant when `backend=revealjs` but arguably someone would want to use another backend named `revealjs`

Reply:
Indeed the `revealjsdir` should be defined in the converter when the environment is Node.js (or here but the path is controlled by the converter so it makes more sense to define the attribute in the converter)